### ISSUE TITLE
Chem.MolToSmarts param misnomer

### ISSUE THREAD

**Issue by elcaceres** [rdkit/rdkit#792]
Chem.MolToSmarts has a parameter named isomericSmarts (bool) in the [docs](http://www.rdkit.org/Python_Docs/rdkit.Chem.rdmolfiles-module.html#MolToSmarts), but the function asks for isomericSmiles (bool) instead.

**Comment by bp-kelley** [comment]
thanks, the signature is correct

```
MolToSmarts( (Mol)mol [, (bool)isomericSmiles=False]) -> str :
```

But the docs are indeed wrong:

```
      ARGUMENTS:
    
        - mol: the molecule
        - isomericSmarts: (optional) include information about stereochemistry in
          the SMARTS.  Defaults to false.
```

**Comment by greglandrum** [comment]
Fixed by  cb0515a
